test/index.ts

```diff
@@ -30,13 +30,14 @@ import {grpc} from 'google-gax';
 import * as sinon from 'sinon';
 import * as spnr from '../src';
 import {Duplex} from 'stream';
-import {CreateInstanceRequest} from '../src/index';
+import {CreateInstanceRequest, CreateInstanceConfigRequest} from '../src/index';
 import {
   GetInstanceConfigOptions,
   GetInstanceConfigsOptions,
   GetInstancesOptions,
 } from '../src';
 import {CLOUD_RESOURCE_HEADER} from '../src/common';
+import * as inst from '../src/instance';
```

Comment:
(Just to verify): Do we need this entire import?

Reply:
This is an unused import. Will remove this. Thanks for pointing this out.